srcs/http/response/http_serverinfo_check/http_serverinfo_check.cpp

```diff
@@ -19,15 +22,18 @@ HttpServerInfoCheck::Check(const MockDtoServerInfos &server_info, HttpRequestFor
 void HttpServerInfoCheck::CheckDTOServerInfo(
 	CheckServerInfoResult    &result,
 	const MockDtoServerInfos &server_info,
-	HeaderFields             &header_fields
+	const HeaderFields       &header_fields
 ) {
-	if (static_cast<size_t>(std::atoi(header_fields["Content-Length"].c_str())) >
-		server_info.client_max_body_size) { // Check content_length
-		throw HttpException("Error: payload too large.", PAYLOAD_TOO_LARGE);
-	} else if (!server_info.error_page.second.empty()) { // Check error_page
-		result.error_page = server_info.error_page;
+	if (header_fields.find(CONTENT_LENGTH) != header_fields.end()) {
+		utils::Result<unsigned int> content_length =
+			utils::ConvertStrToUint(header_fields.at(CONTENT_LENGTH));
```

Comment:
`http_parse.cpp` では size_t に変換してるので、content-length は統一して size_t が良いのかもです
```suggestion
		utils::Result<std::size_t> content_length =
			utils::ConvertStrToSize(header_fields.at(CONTENT_LENGTH));
```